src/devrules/cli_commands/dashboard.py

```
@@ -30,33 +34,31 @@ def dashboard(
+                "Dashboard requires the 'tui' dependency group.",
             )
-            typer.echo("\nInstall with:")
-            typer.secho("  pip install devrules[tui]", fg=typer.colors.CYAN)
-            typer.echo("\nOr if using uv:")
-            typer.secho("  uv pip install devrules[tui]", fg=typer.colors.CYAN)
-            raise typer.Exit(code=1)
+            prompter.info("Install with:")
+            prompter.indented_message("pip install devrules[tui]")
+            prompter.info("Or if using uv:")
+            prompter.indented_message("uv pip install devrules[tui]")
+            raise prompter.exit(code=1)
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)